kmapper/kmapper.py

```diff
@@ -827,6 +827,63 @@ def data_from_cluster_id(self, cluster_id, graph, data):
         else:
             return np.array([])
 
+    def clusters_from_cover(self, cube_ids, graph):
+        """Returns the clusters and their members from the subset of the cover spanned by the given cube_ids
```

Comment:
Thinking out loud. I'm trying to think of another name. Kmapper has a separate `Cover` class, so calling this `clusters_from_cover` suggests to me that a cover should be passed, but it isn't.

But a Cover doesn't have clusters, so I don't think this should go in the `Cover` class.

If `graph` were a class, this would go in there as `graph.find_clusters_by_cube_ids(cube_ids)` or something.

Sort-of following the pattern from the last PR, maybe we rename this to `find_clusters` <--